Source/Rhetos.Dsl/MacroOrderRepository.cs

```diff
@@ -79,21 +80,160 @@ private string ReportMacroOrders(IEnumerable<MacroOrder> macroOrders)
         /// <param name="macroOrders">Tuple's Item1 is EvaluatorName, Item2 is EvaluatorOrder.</param>
         public void Save(IEnumerable<MacroOrder> macroOrders)
         {
-            _saveOrderLogger.Trace(() => "\r\n" + ReportMacroOrders(macroOrders));
-            string sql = string.Join("\r\n", macroOrders.Select(GetSaveSql));
-            _sqlExecuter.ExecuteSql(new[] { sql });
+            string serializedConcepts = JsonConvert.SerializeObject(macroOrders);
+            string path = Path.Combine(Paths.GeneratedFolder, MacroOrderFileName);
+            File.WriteAllText(path, serializedConcepts, Encoding.UTF8);
         }
 
-        private string GetSaveSql(MacroOrder macroOrder)
+        private List<MacroOrder> GetDefaultOrder()

```

Comment:
I think we should remove the initial ordering from the source (`GetDefaultOrder`), even though it was previously present.
The performance degradation of the first deployment is not that important, and it feels wrong to maintain this data as it contains information from plugins that should never be referenced in the core framework.